backend/fms_core/viewsets/sample.py

```diff
@@ -416,19 +417,17 @@ def versions(self, _request, pk=None):
 
     @action(detail=False, methods=["post"])
     def add_samples_to_study(self, request, pk=None):
-        excepted_sample_ids = request.data.get("excepted_sample_ids")
         default_selection = request.data.get("default_selection", False)
```

Comment:
I would change the name to something more frontend agnostic ... The default being unclear in this case. Maybe call it "negative_selection" indicating that you remove the ids listed from the selection ?